python/libcudf/pyproject.toml

```diff
@@ -53,7 +53,8 @@ libcudf = "libcudf"
 select = [
     "distro-too-large-compressed",
 ]
-
+# Default precision is 2. Bump this a bit for better visibility.
+output_file_size_precision = 4
```

Comment:
Trying [the config option]( https://github.com/jameslamb/pydistcheck/blob/4afb495763938471721f2c1fa7c1b86ea02fc339/docs/_static/defaults.toml#L41-L43 ) instead

Reply:
Well that didn't work either. 

Here is [the error from CI]( https://github.com/rapidsai/cudf/actions/runs/14923303573/job/41922649218?pr=18731#step:11:5843 ):

```python
ValueError: Configuration value 'output_file_size_precision' is not recognized by pydistcheck
```